### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Bonsen** [Gelu74/DSB_2018#3]
If I use 4 gpus,:
InvalidArgumentError (see above for traceback): Integer division by zero
	 [[Node: training/SGD/gradients/mrcnn_bbox_loss_1/concat_grad/mod = FloorMod[T=DT_INT32, _class=["loc:@mrcnn_bbox_loss_1/concat"], _device="/job:localhost/replica:0/task:0/device:CPU:0"](split_2/split_dim, training/SGD/gradients/mrcnn_bbox_loss_1/concat_grad/Rank)]]
	 [[Node: training/SGD/gradients/tower_3/mask_rcnn/res4t_branch2a/BiasAdd_grad/BiasAddGrad/_13835 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:3", send_device_incarnation=1, tensor_name="edge_76975_training/SGD/gradients/tower_3/mask_rcnn/res4t_branch2a/BiasAdd_grad/BiasAddGrad", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

tensorflow 1.4
keras 2.1
